packages/vanilla/src/components/pagination/pagination.md

```diff
@@ -0,0 +1,36 @@
+# Panel
+
+## Purpose
+
+Pagination is used for splitting up content or data into several pages with a control for navigating to the next or previous page. 
+
```

Comment:
A Purpose should be one line. The following 2 lines are part of the functional requirements.